### ISSUE TITLE
Cannot find module `express` in the v0.2.0 version

### ISSUE THREAD

**Issue by HardSource** [rally25rs/k8s-when-ready#29]
The installation of dependencies seems not to have been installed correctly for version [v0.2.0](https://github.com/rally25rs/k8s-when-ready/releases/tag/v0.2.0). 
When running `node index.js` the following error is thrown:

```
node:internal/modules/cjs/loader:1147
  throw err;
  ^

Error: Cannot find module 'express'
Require stack:
- /usr/src/app/index.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
    at Module._load (node:internal/modules/cjs/loader:985:27)
    at Module.require (node:internal/modules/cjs/loader:1235:19)
    at require (node:internal/modules/helpers:176:18)
    at Object.<anonymous> (/usr/src/app/index.js:1:17)
    at Module._compile (node:internal/modules/cjs/loader:1376:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
    at Module.load (node:internal/modules/cjs/loader:1207:32)
    at Module._load (node:internal/modules/cjs/loader:1023:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/usr/src/app/index.js' ]
}

Node.js v20.10.0
```

**Comment by rally25rs** [comment]
Good catch, thanks for noticing this.

Sorry it took a while to follow up on this. (Hundreds of GitHub notifications a day between open source projects and my day-job, so this one slipped under the radar...)

It looks like in the upgrade from yarn v1 to v4 I forgot to account for it using yarn PnP now.

**Comment by rally25rs** [comment]
✅ This is now fixed and new images published to DockerHub as version `0.2.1`

**Comment by HardSource** [comment]
Thank you very much @rally25rs 🙌
